### PR TITLE
Apxg load userdata

### PR DESCRIPTION
**Pull request by dmaddoxwhite**
Add methods to read user data csv files and save as json into project folder. Then read json files into main standards data array.

### CODE REVIEW

File: lib/openstudio-standards/standards/ashrae_90_1_prm/userdata_csv/userdata_wateruse_equipment.csv [hunk 1]
```diff
@@ -0,0 +1 @@
+name,bulding_type_swh�
```
Comment (lymereJ) [inline lib/openstudio-standards/standards/ashrae_90_1_prm/userdata_csv/userdata_wateruse_equipment.csv:1]: Is that strange character expected?

File: lib/openstudio-standards/standards/ashrae_90_1_prm/userdata_csv/userdata_airloop_hvac.csv [hunk 2]
```diff
@@ -0,0 +1 @@
+name,has_fan_power_credit_fully_ducted,has_fan_power_credit_return_or_exhaust_flow_control�,has_fan_power_credit_exhaust_treatment ,has_fan_power_credit_filtration_m9to12,has_fan_power_credit_filtration_m13to15,clean_filter_pressure_drop_for_fan_power_credit_filtration_m16plus,fan_power_credit_gas_phase_cleaners,fan_power_credit_biosafety,fan_power_credit_other_than_coil_runaround,has_fan_power_credit_coil_runaround,fan_power_credit_evaporative_humidifier_or_cooler,has_fan_power_credit_sound_attenuation,has_fan_power_credit_exhaust_serving_fume_hoods,has_fan_power_credit_lab_or_vivarium_highrise_vertical_duct,economizer_exception_for_gas_phase_air_cleaning,economizer_exception_for_open_refrigerated_cases,exhaust_energy_recovery_exception_for_toxic_fumes_etc,exhaust_energy_recovery_exception_for_type1_kitchen_hoods,exhaust_energy_recovery_exception_for_type_distributed_exhaust ,exhaust_energy_recovery_exception_for_dehumidifcation_with_series_cooling_recovery
```
Comment (lymereJ) [inline lib/openstudio-standards/standards/ashrae_90_1_prm/userdata_csv/userdata_airloop_hvac.csv:1]: Do we need to stage these files? Some of the fields include strange characters.

File: test/90_1_prm/test_appendix_g_prm.rb [hunk 3]
```diff
@@ -68,6 +72,11 @@ def generate_prototypes(prototypes_to_generate)
         end
       end
 
+      # user data CSV files are in @user_data_dir, if appicable
+      # user data JSON files will be created in sub-folder inside @test_dir
+      # json_path = @prototype_creator.convert_userdata_csv_to_json(@user_data_dir, @test_dir)
+
+
```
Comment (lymereJ) [inline test/90_1_prm/test_appendix_g_prm.rb:79]: Do we want to keep these comments?

File: test/90_1_prm/test_appendix_g_prm.rb [hunk 4]
```diff
@@ -111,6 +120,13 @@ def generate_baseline(prototypes_generated, id_prototype_mapping)
       # Initialize Standard class
       @prototype_creator = Standard.build('90.1-PRM-2019')
 
+      # user data CSV files are in @user_data_dir, if appicable
+      # user data JSON files will be created in sub-folder inside @test_dir
+      model_name = mod.empty? ? "#{building_type}-#{template}-#{climate_zone}" : "#{building_type}-#{template}-#{climate_zone}-#{mod_str}"
+      proto_run_dir = "#{@test_dir}/#{model_name}"
+      json_path = @prototype_creator.convert_userdata_csv_to_json(@user_data_dir, proto_run_dir)
+      @prototype_creator.load_userdata_to_standards_database(json_path)
```
Comment (lymereJ) [inline test/90_1_prm/test_appendix_g_prm.rb:128]: Do we want to test that the data has been load correctly?

**Review by lymereJ — APPROVED** [review]
Looks good! I did left a few comments, let me know what you think, thanks!

File: lib/openstudio-standards/standards/ashrae_90_1_prm/userdata_csv/userdata_wateruse_equipment.csv (hunk 1 above)
Comment (dmaddoxwhite) [reply lib/openstudio-standards/standards/ashrae_90_1_prm/userdata_csv/userdata_wateruse_equipment.csv:1]: No, I will remove it.

File: lib/openstudio-standards/standards/ashrae_90_1_prm/userdata_csv/userdata_airloop_hvac.csv (hunk 2 above)
Comment (dmaddoxwhite) [reply lib/openstudio-standards/standards/ashrae_90_1_prm/userdata_csv/userdata_airloop_hvac.csv:1]: I did a search and hopefully got all of the bad chars.

File: test/90_1_prm/test_appendix_g_prm.rb (hunk 3 above)
Comment (dmaddoxwhite) [reply test/90_1_prm/test_appendix_g_prm.rb:79]: No, I'll remove them.

File: test/90_1_prm/test_appendix_g_prm.rb (hunk 4 above)
Comment (dmaddoxwhite) [reply test/90_1_prm/test_appendix_g_prm.rb:128]: I ran a temporary test to make sure it works. We will add permanent tests when we get to JIRA tasks that access the user data.

**Pull request merged**